apps/core/lib/core/data/model/stock/stock_item.ex

```diff
@@ -1,8 +1,8 @@
-defmodule Core.Snitch.Data.Model.Stock.StockItem do
+defmodule Snitch.Core.Data.Model.Stock.StockItem do
```

Comment:
Shouldn't this be `Snitch.Core.Data.Model.StockLocation` as per [my comment](https://github.com/aviabird/snitch/issues/15#issuecomment-369660600)? If so, the namespace for all `Snitch.Core.Data.*.Stock.*` should be changed. 
> This might also make the `using` macro in `*/stock.ex` redundant.

Reply:
Yes, but didn't want to take the risk, because it was done by @pkrawat1 and I did not want to mess up anybody else's code.

Reply:
You can do it. Ask for reviews from @pkrawat1 later. Simple!

Reply:
Change this as per @oyeb's comments.